src/Components/Comments.jsx

```diff
@@ -20,21 +20,23 @@ function Comments({article_id}) {
         })
     }, []);
 
+      
     if (isLoading) {
         return (
             <p className="loading-state">Loading...</p>
         )
     } else {
         return (
             <>
-            <h3>Comments:</h3>
+            <br></br><br></br><br></br><br></br>
```

Comment:
I don't hate adding breaks like this really, but I suppose this should be done with CSS.